src/python/pants/backend/python/util_rules/pex_from_targets.py

```diff
@@ -104,6 +106,10 @@ def __init__(
             interpreter constraints to not be used because platforms already constrain the valid
             Python versions, e.g. by including `cp36m` in the platform string.
         :param additional_args: Any additional Pex flags.
+        :param additional_args: Any additional Pex flags that should be used with the lockfile.pex.
```

Comment:
```suggestion
        :param additional_lockfile_args: Any additional Pex flags that should be used with the lockfile.pex.
```

Reply:
Oops, missed by 43 seconds! Will fix in a related followup.